pages/attacks/Embedding_Null_Code.md

```diff
@@ -14,65 +14,53 @@ permalink: /attacks/Embedding_Null_Code
 
 The Embedding NULL Bytes/characters technique exploits applications that
 don’t properly handle postfix NULL terminators. It is used as a
-technique to perform other attacks, like directory browsing, path
-traversal, SQL injection, execution of arbitrary code, among others. It
-can be found lots of vulnerable applications and exploits available to
-abuse systems using this technique.
+technique to perform other attack such as directory browsing, path
```

Comment:
attacks should still be plural

Reply:
Fixed, though I made another adjustment to the start of the sentence.